### PR TITLE
Make iso properties non modal.

### PR DESCRIPTION
**Pull request by RachelBryk**

### CODE REVIEW

File: Source/Core/DolphinWX/ISOProperties.cpp [hunk 1]
```diff
@@ -675,8 +661,10 @@ void CISOProperties::OnClose(wxCloseEvent& WXUNUSED (event))
 {
 	if (!SaveGameConfig())
 		WxUtils::ShowErrorDialog(wxString::Format(_("Could not save %s."), GameIniFileLocal.c_str()));
-
-	EndModal(bRefreshList ? wxID_OK : wxID_CANCEL);
+	if (bRefreshList)
+		((CGameListCtrl*)GetParent())->Update();
+	Destroy();
+	//EndModal(bRefreshList ? wxID_OK : wxID_CANCEL);
```
Comment (lioncash) [inline Source/Core/DolphinWX/ISOProperties.cpp]: Can just get rid of EndModal

File: Source/Core/DolphinWX/Cheats/CreateCodeDialog.cpp [hunk 2]
```diff
@@ -20,9 +20,10 @@
 // Fired when an ActionReplay code is created.
 wxDEFINE_EVENT(UPDATE_CHEAT_LIST_EVENT, wxCommandEvent);
 
-CreateCodeDialog::CreateCodeDialog(wxWindow* const parent, const u32 address)
+CreateCodeDialog::CreateCodeDialog(wxWindow* const parent, const u32 address, std::vector<ActionReplay::ARCode> _arCodes)
```
Comment (lioncash) [inline Source/Core/DolphinWX/Cheats/CreateCodeDialog.cpp]: You can pass the vector by reference.

File: Source/Core/DolphinWX/ARCodeAddEdit.cpp [hunk 3]
```diff
@@ -26,12 +26,12 @@
 #include "Core/ActionReplay.h"
 #include "Core/ARDecrypt.h"
 #include "DolphinWX/ARCodeAddEdit.h"
-#include "DolphinWX/ISOProperties.h"
 #include "DolphinWX/WxUtils.h"
 
-CARCodeAddEdit::CARCodeAddEdit(int _selection, wxWindow* parent, wxWindowID id, const wxString& title, const wxPoint& position, const wxSize& size, long style)
+CARCodeAddEdit::CARCodeAddEdit(int _selection, std::vector<ActionReplay::ARCode> _arCodes, wxWindow* parent, wxWindowID id, const wxString& title, const wxPoint& position, const wxSize& size, long style)
```
Comment (lioncash) [inline Source/Core/DolphinWX/ARCodeAddEdit.cpp]: Can pass by const reference.

File: Source/Core/DolphinWX/PatchAddEdit.h [hunk 4]
```diff
@@ -25,7 +25,8 @@ class wxWindow;
 class CPatchAddEdit : public wxDialog
 {
 public:
-	CPatchAddEdit(int _selection, wxWindow* parent,
+	CPatchAddEdit(int _selection, std::vector<PatchEngine::Patch> _onFrame,
```
Comment (lioncash) [inline Source/Core/DolphinWX/PatchAddEdit.h]: Can pass the vector by const reference.

**Comment by JMC47** [comment]
Everything seems to behave as expected.  It may confuse users that they have to close the game properties window in order for the settings to change, but, meh.

**Pull request merged**